editor/enable-tracking-prompt/test/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
```

Comment:
`shallow` doesn't work here because we reach inside a `Button` element and grab its text.  Open to other suggestions.